ansible/roles/master/defaults/main.yml

```diff
@@ -1,7 +1,15 @@
-kube_master_insecure_port: 8080
```

Comment:
If unset, the apiserver defaults the insecure port to 8080. I added it in so it could be tweaked more easily.